CHANGELOG.rst

```diff
@@ -4,6 +4,17 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
+1.0.0-alpha.5-SNAPSHOT
```

Comment:
```suggestion
1.0.0-alpha.5-SNAPSHOT (2021-03-17)
```
We will need that later for the qube linting and to keep it consistent with previous designs